_includes/sqlstyle.guide.md

```diff
@@ -170,22 +151,6 @@ spacing is used. Do not crowd code or remove natural language spaces.
 
 #### Spaces
 
-Spaces should be used to line up the code so that the root keywords all end on
-the same character boundary. This forms a river down the middle making it easy for
-the readers eye to scan over the code and separate the keywords from the
-implementation detail. Rivers are [bad in typography][rivers], but helpful here.
```

Comment:
I think this thing about the river of whitespace is really unhelpful, which is why i deleted it.  

the reason it's unhelpful is that is privileges things like `AND` and `OR` at the same level as `SELECT`, `WHERE`, etc, and I think that's counterproductive. 

As far as the alternate, i did kind of write up an alternate down in the "indentation" section